components/ws-proxy/pkg/proxy/routes_test.go

```diff
@@ -662,7 +662,7 @@ func TestRoutes(t *testing.T) {
 				Header:       "",
 			}
 
-			proxy := NewWorkspaceProxy(ingress, cfg, router, &fakeWsInfoProvider{infos: workspaces})
+			proxy := NewWorkspaceProxy(ingress, cfg, router, &fakeWsInfoProvider{infos: workspaces}, nil)
```

Comment:
Adding a test case will be good

Reply:
Why do you need to generate everything for unit test? You can just pass some random strings and check that you get them when you call a route? As we do for other routes

Reply:
Can we change code that it is easy to test fro routes?